components/Footer/index.js

```diff
@@ -4,18 +4,33 @@ import styled from "styled-components";
 import Link from "next/link";
 
 export default function Footer({ currentPath }) {
+  const router = useRouter();
+
+  const handleHomeClick = () => {
+    if (router.pathname === "/") {
+      router.reload();
+    } else {
+      router.push("/");
+    }
+  };
+
   return (
     <StyledFooter>
       <LinkList>
         <NavLink>
```

Comment:
Semantically speaking, this would be a `ListItem` rather than a `NavLink`, because it isn't a link itself, it just contains one.

Reply:
Thank you. All updated.